pkg/cmd/admin/policy/remove_from_project.go

```diff
@@ -109,8 +121,24 @@ func (o *RemoveFromProjectOptions) Run() error {
 			if err != nil {
 				return err
 			}
+
+			if diff := existingGroups.Difference(currBinding.Groups); len(diff) != 0 {
+				fmt.Fprintf(o.Out(), "Removing %s/%s from groups %v in project %s.\n", currBinding.RoleRef.Namespace, currBinding.RoleRef.Name, diff.List(), o.BindingNamespace)
+				groupsRemoved.Insert(diff.List()...)
+			}
+			if diff := existingUsers.Difference(currBinding.Users); len(diff) != 0 {
+				fmt.Fprintf(o.Out(), "Removing %s/%s from users %v in project %s.\n", currBinding.RoleRef.Namespace, currBinding.RoleRef.Name, diff.List(), o.BindingNamespace)
+				usersRemoved.Insert(diff.List()...)
+			}
 		}
 	}
 
+	if diff := util.NewStringSet(o.Groups...).Difference(groupsRemoved); len(diff) != 0 {
+		fmt.Fprintf(o.Out(), "Groups %v did not have access to project %s.\n", diff.List(), o.BindingNamespace)
```

Comment:
Be careful how you word that... if they have cluster bindings, they do have access. Maybe something like "Groups %v were not bound to roles in project %s.\n"?